docs/src/api/class-browsertype.md

```diff
@@ -62,6 +62,11 @@ This methods attaches Playwright to an existing browser instance.
   - `logger` <[Logger]> Logger sink for Playwright logging. Optional.
   - `timeout` <[float]> Maximum time in milliseconds to wait for the connection to be established. Defaults to
     `30000` (30 seconds). Pass `0` to disable timeout.
+  - `protocol` <"playwright"|"cdp"> To use a Chrome DevTools Protocol endpoint, pass "cdp". Defaults to "playwright".
+
+:::note
+Connecting over the Chrome DevTools Protocol is only supported for Chromium-based browsers.
```

Comment:
Why don't we put this in the option description, to not draw unneeded attention?